Makefile.include

```diff
@@ -130,6 +130,14 @@ endif
 include $(RIOTBASE)/Makefile.modules
 include $(RIOTBOARD)/$(BOARD)/Makefile.include
 include $(RIOTCPU)/$(CPU)/Makefile.include
+
+# get number of interfaces straight before resolving dependencies
+NG_NETIF_NUMOF ?= 1
+
+ifneq ($(NG_NETIF_NUMOF),1)
+  CFLAGS += -DNG_NETIF_NUMOF=$(NG_NETIF_NUMOF)
+endif
```

Comment:
Well, of course they can, but I assumed that there is a reason to get this number that early in the build system.